components/automate-chef-io/content/docs/backup.md

```diff
@@ -305,5 +224,11 @@ y
 Success: Backups deleted
 ```
 
+Chef Automate does not provide a mechanism to prune all but a certain number of the most recent backups. You can manage this manually by parsing the output of `chef-automate backup list` and applying the command `chef-automate backup delete`. For example:
+
+```bash
```

Comment:
If we don't want to include the example, that's fine. I included it because it seems too small for a blog post.